### PR TITLE
Contacts flow

### PR DESCRIPTION
**Pull request by cryptosalomao**

### CODE REVIEW

**Review by zupzup — APPROVED** [review]
LGTM :+1:

File: src/pages/contacts/Create.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+      <div
+        className={`flex flex-col gap-3 ${
+          isExtraFieldsVisible ? "block" : "hidden"
+        }`}
+      >
+        <Input id="dateOfBirth" label="Date of birth (MM/DD/YYYY" />
+        <Select>
+          <SelectTrigger label="Country of issuing" id="countryOfIssuing">
+            <SelectValue placeholder="Select an option" />
```
Comment (mjkeaton) [inline src/pages/contacts/Create.tsx:80]: Should placeholders already be localized with `FormattedMessage`?

```suggestion
            <SelectValue placeholder={<FormattedMessage
            id="pages.createContact.select.country"
            defaultMessage="Select an option"
            description="Placeholder for country selection"
          />} />
```

This applies to other non-translated fields/placeholders/etc. in this file.

Comment (mtbitcr) [reply src/pages/contacts/Create.tsx:80]: @mjkeaton we do changes in our i18n format now. We use other id format. Check https://github.com/BitcreditProtocol/E-Bill-frontend/pull/85

**Review by mjkeaton — APPROVED** [review]
There is manual work necessary when reviewing the screens. It would be nice if the page routes are registered so that one can browse them without changing the code for the review process. e.g. reaching the empty list screen like for notifications (`/notifications-empty`) or bills (`/bills-empty`). What do you think?

Other than that: Looks nice. Approved. :muscle:

**Comment by mtbitcr** [comment]
> There is manual work necessary when reviewing the screens. It would be nice if the page routes are registered so that one can browse them without changing the code for the review process. e.g. reaching the empty list screen like for notifications (`/notifications-empty`) or bills (`/bills-empty`). What do you think?
> 
> Other than that: Looks nice. Approved. 💪

agree

File: src/pages/contacts/Create.tsx (hunk 1 above)
Comment (cryptosalomao) [reply src/pages/contacts/Create.tsx:80]: Good catch, thank you! I'm gonna replace that. In this case, we must use the imperative API so we don't have to change the typedef for the component.

Ref: https://formatjs.github.io/docs/react-intl/api/#useintl-hook

**Comment by cryptosalomao** [comment]
> There is manual work necessary when reviewing the screens. It would be nice if the page routes are registered so that one can browse them without changing the code for the review process. e.g. reaching the empty list screen like for notifications (`/notifications-empty`) or bills (`/bills-empty`). What do you think?
> 
> Other than that: Looks nice. Approved. 💪

We can do that with Storybook. Will add it to the PR. Thanks!

**Pull request merged**